apps/content/models.py

```
@@ -380,6 +381,19 @@ class AssetVersion(DeleteFilesOnDeleteMixin, BaseModel):
 
+    # is_active = models.BooleanField(default=True)
+    #
+    # class Meta:
+    #     unique_together = [("name", "asset")]
+
+    def save(self, *args, **kwargs):
+        if self.file_url and not self.size_bytes:
+            try:
+                self.size_bytes = self.file_url.size
+            except Exception:
+                pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)